### PR TITLE
Test 

### PR DESCRIPTION
**Pull request by haipq7641**

### CODE REVIEW

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.framgia.androidtestci;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L4
 

Wrong order for android.os.Bundle import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        // test
+    }
+
+    private void TESTphatxem() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    }
+
+        private void ko_loi_a() {
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L19
 

&apos;method def modifier&apos; have incorrect indentation level 8, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            String ref = "refs/pull/1/merge";
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;member def type&apos; have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+            System.out.println("  ref  --> " + ref);
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L21
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+            ref = ref.replace("refs/pull/","").replace("/merge","");
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;,&apos; is not followed by whitespace.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;,&apos; is not followed by whitespace.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        }
```
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L23
 

&apos;method def rcurly&apos; have incorrect indentation level 8, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 1 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L4
 

Wrong order for android.os.Bundle import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L4
 

Wrong order for android.os.Bundle import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 2 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L15
 

Method names should start with a lower case letter (e.g. getWidth())

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 3 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L19
 

&apos;method def modifier&apos; have incorrect indentation level 8, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 4 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;member def type&apos; have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L20
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 5 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L21
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 6 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;method def&apos; child have incorrect indentation level 12, expected level should be 8.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;,&apos; is not followed by whitespace.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L22
 

&apos;,&apos; is not followed by whitespace.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 7 above)
Comment (KaKaVip) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
Severity: ERROR
File: /Users/pham.quy.hai/Develop/AndroidTestCI/app/src/main/java/com/framgia/androidtestci/MainActivity.java L23
 

&apos;method def rcurly&apos; have incorrect indentation level 8, expected level should be 4.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

File: app/src/main/java/com/framgia/androidtestci/MainActivity.java (hunk 1 above)
Comment (haipq7641) [inline app/src/main/java/com/framgia/androidtestci/MainActivity.java:4]: Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
Severity: ERROR
File: /Users/haipq/Develop/WorkSpace/CI-CD/android-test-ci/app/src/main/java/com/framgia/androidtestci/MainActivity.java L4

Wrong order for android.os.Bundle import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>

**Pull request merged**